source/elements/oneDAL/source/glossary.rst

```diff
@@ -13,92 +13,113 @@ Machine learning terms
     Categorical feature
         A :term:`feature` with a discrete domain without ordering between values.
```

Comment:
to double-check: can categorical feature be ordinal, thus, ordinal and nominal are two classes of categorical features.  Also, double-check with Mikhail A. on another interval(?) class of features

Reply:
Thanks, that was a mistake. Added terms for nominal, interval and ratio features